contents/docs/surveys/_snippets/ios-surveys-installation.mdx

```diff
@@ -0,0 +1,68 @@
+## Step one: Add PostHog to your app
+
+> 🚧 **Note:** Mobile surveys for iOS is currently in **beta**. We'd love to hear your feedback on betas via the [in-app support panel](https://us.posthog.com#panel=support%3Afeedback%3Asurveys%3Alow%3Atrue) or one of our other [support options](/docs/support-options).
+
+> **Note:** Using surveys requires PostHog's iOS SDK version >= 3.22.0. It's recommended to always use the latest version.
+
+import IOSInstall from "../../integrate/_snippets/install-ios.mdx"
+
+<IOSInstall />
+
+
+
+## Step two: Enable surveys in your configuration
+
+> **⚠️ Caution:**
+> - Surveys are currently in beta and only accessible through the `Experimental` SPI in Swift.
+> - We recommend thorough testing in your development environment before deploying to production.
+> - Survey APIs may change frequently and introduce breaking changes between SDK releases. Consider this when planning updates.
+
+To enable surveys in your iOS app:
+
+1. First, import PostHog with access to its experimental APIs:
+```swift
+@_spi(Experimental) import PostHog // Import with Experimental SPI
```

Comment:
tbh I think this just adds friction, if the feature is disabled by default, this is just an extra step

Reply:
It's fair point and i defer it to you.
Other frameworks just have a comment, as long as its disabled by default and comments/docs tell its beta, i think the intention is clear.
If the spi experimental is common on iOS, then go for it